docs/resources/client/models/modelloaders.md

```diff
@@ -295,7 +294,8 @@ public class MyDynamicModel implements IDynamicBakedModel {
 
     @Override
     public TextureAtlasSprite getParticleIcon() {
-        // Return MISSING_TEXTURE.sprite() if you don't need a particle, e.g. when in an item model context.
+        // Return 'new Material(TextureAtlas.LOCATION_BLOCKS, MissingTextureAtlasSprite.getLocation()).sprite()'
```

Comment:
This change shouldn't be necessary, MISSING_TEXTURE is defined previously in that class.

Reply:
I'm blind.